demisto_sdk/commands/common/hook_validations/conf_json.py

```diff
@@ -104,6 +104,8 @@ def has_unskipped_test_playbook(self, current_file, entity_id, file_path, test_p
         Retrun:
             True if the content entity has at least one unskipped test playbook.
         """
+        if test_playbook_ids is None:
+            test_playbook_ids = []
```

Comment:
Why in the function declaration you change the default to `None` instead of `[]` and now you change it back?

Reply:
It was recommended by the IDE - if you set the default value to [] and someone alters the list, the default value itself changes. Defining the default as None and then reassigning is a safer option